dcc-portal-api/src/main/java/org/icgc/dcc/portal/repository/TermsLookupRepository.java

```diff
@@ -21,6 +21,7 @@
 import static com.google.common.base.Throwables.propagate;
 import static java.lang.Math.min;
 import static lombok.AccessLevel.PRIVATE;
+import static org.dcc.portal.pql.meta.Type.*;
```

Comment:
Need to update your settings to resolve these. Otherwise we may have a higher chance of conflicts in the future.